services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/proxy.py

```diff
@@ -79,12 +80,18 @@ def get_dynamic_proxy_spec(
             f"traefik.http.routers.{scheduler_data.proxy_service_name}.priority": "10",
             f"traefik.http.routers.{scheduler_data.proxy_service_name}.rule": f"hostregexp(`{scheduler_data.node_uuid}.services.{{host:.+}}`)",
             f"traefik.http.routers.{scheduler_data.proxy_service_name}.middlewares": f"{dynamic_sidecar_settings.SWARM_STACK_NAME}_gzip@docker, {scheduler_data.proxy_service_name}-security-headers",
-            "type": ServiceType.DEPENDENCY.value,
             "dynamic_type": "dynamic-sidecar",  # tagged as dynamic service
-            "study_id": f"{scheduler_data.project_id}",
-            "user_id": f"{scheduler_data.user_id}",
-            "uuid": f"{scheduler_data.node_uuid}",  # needed for removal when project is closed
-        },
+        }
+        | StandardSimcoreDockerLabels(
+            user_id=scheduler_data.user_id,
+            project_id=scheduler_data.project_id,
+            node_id=scheduler_data.node_uuid,
+            product_name=scheduler_data.product_name,
+            simcore_user_agent=scheduler_data.request_simcore_user_agent,
+            swarm_stack_name=dynamic_sidecar_settings.SWARM_STACK_NAME,
+            memory_limit=ByteSize(MEMORY_50MB),
+            cpu_limit=float(CPU_10_PERCENT) / (1.0 * 10**9),
```

Comment:
May I suggest this one?
```suggestion
            cpu_limit=CPU_10_PERCENT / 1e9,
```